README.md

```diff
@@ -1,368 +1,257 @@
-8. If you click back and forth between the react page links, you can see the rails console
-   log as well as the browser console to see what's going on with regards to server rendering and
-   caching.
-
-# Key Tips
-1. See sample app in `spec/dummy` for how to set this up. See note below on ensuring you 
-   **DO NOT RUN `rails s` and instead     run `foreman start`. 
-2. Test out the different options and study the JSX samples in `spec/dummy/client/app/startup`.
-3. Experiment with changing the settings on the `render_component` helper calls in the ERB files.
-2. The file used for server rendering is hard coded as `generated/server.js`
-   (assets/javascripts/generated/server.js).
-3. The default for rendering right now is `prerender: false`. **NOTE:**  Server side rendering does
-   not work for some components, namely react-router, that use an async setup for server rendering.
-   You can configure the default for prerender in your config.
-4. You can expose either a React component or a function that returns a React component. If you 
-   wish to create a React component via a function, rather than simply props, then you need to set 
-   the property "generator" on that function to true. When that is done, the function is invoked 
-   with a single parameter of "props", and that function should return a React element.
-5. Be sure you can first render your react component client only before you try to debug server
-   rendering!
-4. Open up the HTML source and take a look at the generated HTML and the JavaScript to see what's
-   going on under the covers. Not that when server rendering is turned on, then you'll see the
-   server rendered react components. When server rendering is turned off, then you'll only see
-   the `div` element where the inline JavaScript will render the component. You might also notice
-   how the props you pass (a Ruby Hash) becomes inline JavaScript on the HTML page.
-
-## JavaScript Runtime Configuration
-See this [discussion on JavaScript performance](https://github.com/shakacode/react_on_rails/issues/21).
-The net result is that you want to add this line to your Gemfile to get therubyracer as your default
-JavaScript engine.
-
-```ruby
-gem "therubyracer"
-```
+#### Bootstrap via Webpack HMR Dev Server
+When using the webpack dev server, which does not go through Rails, bootstrap is loaded via the [bootstrap-sass-loader](https://github.com/shakacode/bootstrap-sass-loader) which uses the `client/bootstrap-sass-config.js` file. 
 
-## React 0.13 vs. React 0.14
-The main difference for using react_on_rails is that you need to add additional lines in the webpack config files:
+#### Keeping Custom Bootstrap Configurations Synced
+Because the HMR dev server and Rails each load Bootstrap via a different file, any changes to the way components are loaded in one file must also be made to the other file in order to keep styling consistent between the two.
```

Comment:
This definitely requires a little explanation for what files. This by itself sounds totally scary.

Need to emphasize that it's just the 2 manifest files to keep synchronized and those files will load the same other files, so no additional synchronization is needed.